repoconfig.sh

```diff
@@ -1,6 +1,7 @@
 #! /bin/sh
 # shellcheck disable=SC2034
-NODEJS_VERSION=v16
+NODEJS_VERSION=v20
+NODEJS_ALT_VERSION=v18
```

Comment:
won't this always be the penultimate version? consider,
```suggestion
NODEJS_VERSION_PREV=v18
```

Reply:
`NODEJS_VERSION` is also used to automatically install that version of Node. I don't want to assume we will always want to install the latest LTS, and the alt may indeed be higher for some time.

Reply:
Reverted this change